admin/server/auth/device_code.go

```diff
@@ -145,7 +148,9 @@ func (a *Authenticator) handleUserCodeConfirmation(w http.ResponseWriter, r *htt
 	}
 	userID := claims.OwnerID()
 
-	authCode, err := a.admin.DB.FindPendingDeviceAuthCodeByUserCode(r.Context(), userCode)
+	// Remove "-" from user code
+	dbUserCode := userCode[:4] + userCode[5:]
```

Comment:
The user code might be shorter (it's user input), so it's safer to just do: `userCode = strings.ReplaceAll(userCode, "-", "")`

Reply:
In lines below to this, there are multiple error messages in which `userCode` so keeping it as it is and doing the replace to get a new string

Reply:
not sure if you wanted to use same `userCode` variable or think `ReplaceAll` is more safer (why ?) because I think slicing is more efficient than replacing.

Reply:
I see what you mean, to prevent against wrong user input, makes sense.